plugin/trino-blackhole/src/main/java/io/trino/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -448,4 +448,24 @@ public MaterializedViewFreshness getMaterializedViewFreshness(ConnectorSession s
     {
         return new MaterializedViewFreshness(STALE, Optional.empty());
     }
+
+    @Override
```

Comment:
> Add support for materialized view column comments in engine

Can we change the commit title to "Add support for materialized view column comments in engine and blackhole"?

Reply:
Fixed